Website/src/rendering/src/components/FeaturedSpeakers.tsx

```diff
@@ -1,51 +1,90 @@
-import Link from 'next/link';
-import { Text, Image } from '@sitecore-jss/sitecore-jss-nextjs';
+import { Text, Field, RichText, Image, LinkField, Link } from '@sitecore-jss/sitecore-jss-nextjs';
 import { ComponentProps } from 'lib/component-props';
 import { GraphQLSpeaker } from 'src/types/speaker';
+import React from 'react';
 
 export type FeaturedSpeakersProps = ComponentProps & {
   fields: {
     data: {
+      source: {
+        title: Field<string>;
+        content: Field<string>;
+        callToActionLink: {
+          jsonValue: LinkField;
+        };
+        numberOfSpeakers: Field<string>;
+      };
       item: {
         children: {
           results: GraphQLSpeaker[];
         };
       };
     };
   };
-  params: {
-    NumberOfSpeakers: string;
-  };
 };
 
 const FeaturedSpeakers = (props: FeaturedSpeakersProps): JSX.Element => {
-  const speakers =
-    props.fields.data?.item?.children?.results &&
-    props.fields.data.item.children.results
-      .filter((item) => item.featured.value)
-      .sort()
-      .slice(0, parseInt(props.params.NumberOfSpeakers))
-      .map((speaker, index) => (
-        <Link key={index} href={`/speakers/${speaker.itemName}`} passHref>
-          <a className="grid-item">
-            <Image
-              field={speaker.picture.jsonValue}
-              alt={speaker.name}
-              width={265}
-              height={265}
-              loading="lazy"
-            />
-            <div className="item-details">
-              <Text tag="p" field={speaker.name} />
-            </div>
-          </a>
-        </Link>
-      ));
+  const speakers = props.fields.data.item.children.results
+    .filter((item) => item.featured.value)
+    .sort()
+    .slice(
+      0,
+      parseInt(
+        props.fields.data.source.numberOfSpeakers
+          ? props.fields.data.source.numberOfSpeakers.value
+          : '6'
+      )
+    )
+    .map((speaker, index) => (
+      <a key={index} href={`/speakers/${speaker.itemName}`} className="grid-item">
+        <div className="item-image">
+          <Image
+            field={speaker.picture.jsonValue}
+            alt={speaker.name}
+            width={265}
+            height={265}
+            loading="lazy"
+          />
+        </div>
+        <div className="item-details">
+          <Text tag="strong" field={speaker.name} />
```

Comment:
The strong HTML element is not meant to put thing in bold. It is meant to provide emphasis on a word in a sentence. Please change it to a styled p or div.